src/scss/_shabad-page.scss

```diff
@@ -15,11 +15,11 @@ blockquote {
 
 #metadata {
   align-items: center;
-  border: 1px solid $sttm-light-grey;
+  background-color: #e9e9e9;
```

Comment:
If the current colors we have as variables, doesn't work here, maybe we can add some new colors to it and use a variable here instead of hex code.

List of current colors is at `sttm-web/src/scss/_colours.scss`